front/scripts/test_llm_clients.ts

```diff
@@ -265,12 +272,17 @@ const TEST_CONVERSATIONS: TestConversation[] = [
 
 async function runTest(
```

Comment:
Can you add a comment explaining the context and how to run the test ? (cf command in PR description)